proto/Networking/Requests/Messages/ChangeTeamMessage.proto

```diff
@@ -1,9 +1,9 @@
 syntax = "proto3";
 package POGOProtos.Networking.Requests.Messages;
 
-import "POGOProtos/Inventory/Item/ItemId.proto";
+import "POGOProtos/Enums/TeamColor.proto";
 
 message ChangeTeamMessage {
-	.POGOProtos.Inventory.Item.ItemId item = 1;
-	.POGOProtos.Inventory.Item.ItemId team = 2;
+	.POGOProtos.Enums.TeamColor item = 1;
```

Comment:
First is item, second is team.
Or am I missing something?

Reply:
this ? https://github.com/pogosandbox/pogo-protos/pull/6#discussion_r260477238

Reply:
yes